apps/jenkins/jenkins/jenkins.yaml

```diff
@@ -62,6 +62,7 @@ spec:
         -Djenkins.model.Jenkins.logStartupPerformance=true
         -Dorg.jenkinsci.plugins.durabletask.BourneShellScript.LAUNCH_DIAGNOSTICS=true
         -Dkubernetes.websocket.ping.interval=60000
+        -Dorg.jenkins.plugins.lockableresources.DISABLE_SAVE=true
```

Comment:
performance enhancement in latest release https://github.com/jenkinsci/lockable-resources-plugin#improve-performance